### ISSUE TITLE
Duplex contract Fromconfiguration not working

### ISSUE THREAD

**Issue by saisworld** [castleproject/Windsor#32]
I was trying to create a Duplex client model using the "FromConfiguration" feature provided by castle wcf facility. It works fine when registering the service on client side when at runtime when I try to resolve it, it throws error.

```
DuplexClientModel model = new DuplexClientModel
    {
        Endpoint = WcfEndpoint.ForContract<IServiceWithCallback>()
            .FromConfiguration("IServiceWithCallback_Endpoint")

    }.Callback(callbackService);
```

Error is: WcfClientActivator: Could not find proxy component 7c9e34c6-72c4-44b8-a082-e4e4ad3d7777"

Innerexception: The Address property on ChannelFactory.Endpoint was null. The ChannelFactory's Endpoint must have a valid Address specified.

Client App.Config-

endpoint address="net.tcp://localhost:9898/ServiceWithCallback" binding="netTcpBinding" bindingConfiguration="netTcpBindingConfiguration" contract="Castle.Facilities.WcfIntegration.Tests.Components.IServiceWithCallback" name="IServiceWithCallback_Endpoint" />

Using the same service but using the "BoundTo" and "At" instead of "FromConfiguration" it works just fine.

```
DuplexClientModel model = new DuplexClientModel
    {
        Endpoint = WcfEndpoint.ForContract<IServiceWithCallback>()                 
           .BoundTo(new NetTcpBinding())
           .At("net.tcp://localhost:9898/ServiceWithCallback")

    }.Callback(callbackService);
```

**Comment by fir3pho3nixx** [comment]
No PR, closing.

**Comment by djwelch** [comment]
I actually fixed this issue here

https://github.com/castleproject/Windsor/commit/f520d6c08ccb15609f13e152b28f25a65b4b1da9

Can't find  the pull request I did, no idea why.

**Comment by jonorossi** [comment]
Thanks @djwelch. Fixed in #59 and was released in 3.3.0.
